routes/employees.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express')
 const router = express.Router()
 const employeesController = require('../controllers/employees')
+const createEmployeeController = require('../controllers/employees/create')
+const updateEmployeeController = require('../controllers/employees/update')
+const deleteEmployeeController = require('../controllers/employees/delete')
 
 router.get('/', employeesController.getEmployees)
 
 router.get('/:id', employeesController.getEmployeeById)
 
 router.get('/firstname/:first_name', employeesController.getEmployeeByFirstName)
 
-// router.post('/', employeesController.createEmployee)
+router.post('/', createEmployeeController.createEmployee)
 
-// router.put('/:id', employeesController.updateEmployee)
+router.put('/:id', updateEmployeeController.updateEmployee)
 
-// router.delete('firstname/:first_name', employeesController.deleteEmployee)
+router.delete('firstname/:first_name', deleteEmployeeController.deleteEmployee)
```

Comment:
this should be `'/:id'`  also, not by first name. 
If there are multiple people with the same first name, there is no way to differentiate which account to delete.